### PR TITLE
Add section about tagging scenarios to acceptance-tests.adoc

### PR DESCRIPTION
**Pull request by phil-davis**
so that developers can know what are the tags used in acceptance test features and scenarios.

### CODE REVIEW

File: modules/developer_manual/pages/core/acceptance-tests.adoc [hunk 1]
```diff
@@ -482,6 +482,162 @@ then put them into a `Background:` section. For example:
...
+== Controlling Running Test Scenarios In Different Environments
+
+A feature or test scenario might only be relevant to run on a system-under-test
+that has a particular environment. For example, a particular app enabled.
+
+To allow the test runner script to run the features and scenarios relevant to
+the system-under-test the feature file or individual scenarios are tagged.
+The test runner script can then filter by tags to select the relevant features and/or scenarios.
+
+=== Tagging A Whole Feature Or Single Scenario
```
Comment (individual-it) [inline modules/developer_manual/pages/core/acceptance-tests.adoc]: I would delete that paragraph, that is gherkin knowledge, not owncloud

Comment (phil-davis) [reply modules/developer_manual/pages/core/acceptance-tests.adoc]: Removed and replaced with a link to the Behat docs about tags.

**Comment by settermjd** [comment]
@phil-davis, I added a series of changes in 69386e9, I hope you don't mind. Just seeking to add extra clarity to your excellent changes.

**Comment by phil-davis** [comment]
LGTM to me thanks.
@settermjd @individual-it somebody approve please and we can merge.

**Comment by phil-davis** [comment]
Backports:
`10.1` #629 
`10.0` #628

**Pull request merged**